Sources/LeafKit/LeafSerialize/LeafSerializer.swift

```diff
@@ -42,8 +46,12 @@ internal struct LeafSerializer {
             case .conditional(let c):  try serialize(c)
             case .loop(let loop):      try serialize(loop)
             case .expression(let exp): try serialize(expression: exp)
-            case .import, .extend, .export:
+            case .extend, .export:
                 throw "\(syntax) should have been resolved BEFORE serialization"
+            case .import:
+                if (!self.ignoreUnfoundImports) {
+                    throw "\(syntax) should have been resolved BEFORE serialization"
+                }
```

Comment:
I think we can tidy this up a bit
```suggestion
            case .import:
                if (!self.ignoreUnfoundImports) {
                    fallthrough
                } else {
                    break
                }
            case .extend, .export:
                throw "\(syntax) should have been resolved BEFORE serialization"
```

Reply:
PS this may not compile, if it doesn't we can stick with the original

Reply:
Hi Tim, thanks for suggestion. It compiles fine. At the end I removed the negation from a condition and switched break and fallthrough so it's one operation less.
```swift
case .import:
  if (self.ignoreUnfoundImports) {
     break
  } else {
    fallthrough
  }
case .extend, .export:
  throw "\(syntax) should have been resolved BEFORE serialization"
```